### DISCUSSION TITLE
migrate all info to my file yesql.bd to file in sql db

### DISCUSSION THREAD

**Discussion by ODA669** [OrchardCMS/OrchardCore#13959] [General]
i need to know how i can migrate all information to sql db

**Comment by hishamco** [comment]
Which info you mean?

**Reply by ODA669** [reply]
i want to pass all information the site save in file to yessql for a file in sql because when i delete the file yessql all ican see in page web is delete an i need to onsert in database to sql  serverand achieve see the page with this file

**Reply by hishamco** [reply]
I didn't understand the point, SQLite database is file-based so you can create your own backup simply by copying & pasting the file

**Reply by ODA669** [reply]
i need to pass the database to sql server i dont want to handle sql lite

**Reply by hishamco** [reply]
If you already have SQLIte you can migrate to SQL Server easily, check this [link](https://stackoverflow.com/questions/163079/exporting-from-sqlite-to-sql-server)

**Reply by ODA669** [reply]
i did thanks, but when i will go to change  in appsettings.json to OrcharedCorte.Cms.Web for start the project its showme how it shows me as if there was no page created, do you know i can fix this

![image](https://github.com/OrchardCMS/OrchardCore/assets/60682935/61c47706-2dc1-46d6-8ef7-5e5f9d56a28e)

i wrote this 

{
  "DatabaseProvider": "SqlConnection",
  "ConnectionString": "Data Source=server;Initial Catalog=123456;Trust Server Certificate=true;User ID=server;Password=123456",
  "Default": {
    "TablePrefix": "TB",
    "State": "Running"
  }
}

when i used this {
  "OrchardCore": {
    "ConnectionString": "...",
    "DatabaseProvider": "SqlConnection",
    "Default" : {
      "State": "Uninitialized",
      "TablePrefix": "Default"
     }
  }
}
 its showme  an error _shellDescriptor = await _session.Query<ShellDescriptor>().FirstOrDefaultAsync(); System.NullReferenceException : 'Object reference not set to an instance of an object.'

**Reply by ODA669** [reply]
i have this error Erreur	TS2792	(TS) Cannot find module '../logger'. Did you mean to set the 'moduleResolution' option to 'node', or to add aliases to the 'paths' option?
